### PR TITLE
Currency: Created build script and build_info.json for Hdijupyterutils

### PR DESCRIPTION
**Pull request by aastha-sharma2**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Have you validated script on UBI 9 container
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: h/hdijupyterutils/hdijupyterutils_ubi_9.3.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+#                    It may not work as expected with newer versions of the
+#                    package and/or distribution. In such case, please
+#                    contact "Maintainer" of this script.
+#
+# ---------------------------------------------------------------------------
+
+# Variables
+PACKAGE_NAME=hdijupyterutils
+PACKAGE_VERSION=${1:-0.20.0}
+PACKAGE_URL=https://github.com/jupyter-incubator/sparkmagic.git
```
Comment (shubham-dayma-ibm) [inline h/hdijupyterutils/hdijupyterutils_ubi_9.3.sh:24]: The package name for given url(https://github.com/jupyter-incubator/sparkmagic.git) is sparkmagic. And hdijupyterutils is sub module of it. You can write build script for hdijupyterutils and not for base package

File: h/hdijupyterutils/hdijupyterutils_ubi_9.3.sh [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+# Install dependencies
+yum install -y git gcc gcc-c++ make wget openssl-devel bzip2-devel libffi-devel zlib-devel python-devel python-pip krb5-devel
+
+# Clone the repository
+git clone $PACKAGE_URL
+cd $PACKAGE_PATH
+git checkout $PACKAGE_VERSION
+
+# install necessay dependencies
+pip install .
```
Comment (shubham-dayma-ibm) [inline h/hdijupyterutils/hdijupyterutils_ubi_9.3.sh:38]: Why we didn't handled installation failed case?

File: h/hdijupyterutils/hdijupyterutils_ubi_9.3.sh [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+pip install pytest mock build
+pip install -r requirements.txt
+
+#install
+if ! (pyproject-build) ; then
```
Comment (shubham-dayma-ibm) [inline h/hdijupyterutils/hdijupyterutils_ubi_9.3.sh:43]: Why are we building package?

File: h/hdijupyterutils/hdijupyterutils_ubi_9.3.sh [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    echo "------------------$PACKAGE_NAME:Install_fails-------------------------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | GitHub | Fail |  Install_Fails"
+    exit 1
+fi
```
Comment (shubham-dayma-ibm) [inline h/hdijupyterutils/hdijupyterutils_ubi_9.3.sh:48]: Are there are no test cases to execute?

**Review by shubham-dayma-ibm — CHANGES_REQUESTED** [review]

File: h/hdijupyterutils/hdijupyterutils_ubi_9.3.sh (hunk 1 above)
Comment (shubham-dayma-ibm) [reply h/hdijupyterutils/hdijupyterutils_ubi_9.3.sh:24]: Resolved

File: h/hdijupyterutils/hdijupyterutils_ubi_9.3.sh (hunk 4 above)
Comment (aastha-sharma2) [reply h/hdijupyterutils/hdijupyterutils_ubi_9.3.sh:47]: > Are there are no test cases to execute?

Removed the package build step and focused on executing tests while skipping the failing ones.

**Review by shubham-dayma-ibm — APPROVED** [review]

File: h/hdijupyterutils/build_info.json [hunk 5]
```diff
@@ -0,0 +1,16 @@
+{
+  "maintainer": "aastha-sharma2",
+  "package_name": "hdijupyterutils",
+  "github_url": "https://github.com/hjson/hjson-py.git",
```
Comment (nikhil-kalbande) [inline h/hdijupyterutils/build_info.json:4]: @aastha-sharma2 source url change should reflect here also

**Review by nikhil-kalbande — CHANGES_REQUESTED** [review]

**Pull request merged**